CodeEditModules/Modules/AppPreferences/src/Sections/AccountsPreferences/Accounts/AccountListItem.swift

```diff
@@ -13,7 +13,7 @@ struct AccountListItem: View {
 
     var body: some View {
         HStack {
-            Image(systemName: "xmark.square.fill")
+            Image("vault.fill")
```

Comment:
Please update to use the new `CodeEditSymbols` package. 

Reference:
* #472 
* https://github.com/CodeEditApp/CodeEditSymbols